python/lsst/ctrl/bps/wms/panda/cmd_line_embedder.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+
+class CommandLineEmbedder:
+    """
+    Class embeds static (constant across a task) values into the pipeline execution command line
+    and resolves submission side environment variables
+
+    Parameters
+    ----------
+    config : `~lsst.ctrl.bps.BPSConfig`
+            BPS configuration that includes the list of dynamic (uniques per job)
+            and submission side resolved variables
+
+    """
+    def __init__(self, config):
+        self.leave_placeholder_params = config.get("placeholderParams", ['qgraphNodeId', 'qgraphId'])
+        self.submit_side_resolved = config.get("submitSideResolvedParams", ['USER'])
+
+    def replace_static_parameters(self, cmd_line, lazy_vars):
+        """ Substitutes the lazy parameters in the command line which are static,
+        the same for every job in the workflow and could be defined once. This function offloads the edge

```

Comment:
This comment is too long. I think this means that bps has not enabled the 79 character limit for docstrings in setup.cfg.

Reply:
It would be nice if there was some project-centric place for these common pieces.  We copied it from some place at the beginning, and obviously should do so again.  Feel free to do that on this ticket or I'll do it on my ticket.

Reply:
I just checked with max-doc-length.  There will be several places in regular bps code that need changed because I mistakenly thought that 79-character maximum was only for docstrings and in code comments could be the same length as the code.

Reply:
Made a ticket to update the setup.cfg file and fix the problems (https://jira.lsstcorp.org/browse/DM-22990).